src/test/java/com/slack/kaldb/util/LocalKafkaSeed.java

```diff
@@ -0,0 +1,23 @@
+package com.slack.kaldb.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.charithe.kafka.EphemeralKafkaBroker;
+import com.slack.kaldb.testlib.TestKafkaServer;
+import java.time.Instant;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class LocalKafkaSeed {
```

Comment:
Do we use this class any where in the code? I can't find references to it. Can we remove it?

Reply:
I went ahead and removed this for now. We should figure out how we want to go about adding the tooling for local development.